packages/arb-token-bridge-ui/src/token-bridge-sdk/BridgeTransferStarterFactory.ts

```diff
@@ -51,8 +51,12 @@ export class BridgeTransferStarterFactory {
       destinationChainErc20Address: props.destinationChainErc20Address
     }
 
-    const { isDeposit, isNativeCurrencyTransfer, isSupported, isTeleport } =
-      getBridgeTransferProperties(props)
+    const {
+      isDeposit,
+      isNativeCurrencyTransfer,
+      isSupported,
+      isValidTeleportChainPair
```

Comment:
maybe this should stay as `isTeleport`, because it refers to the transfer type not chain pairs in here I think?

Reply:
Updated